[2024-12-11T13:06:27.333+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-11T13:06:27.365+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: instacart_etl.etl_dim_department manual__2024-12-11T13:06:24.787823+00:00 [queued]>
[2024-12-11T13:06:27.389+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: instacart_etl.etl_dim_department manual__2024-12-11T13:06:24.787823+00:00 [queued]>
[2024-12-11T13:06:27.390+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-11T13:06:27.428+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): etl_dim_department> on 2024-12-11 13:06:24.787823+00:00
[2024-12-11T13:06:27.449+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5985) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-11T13:06:27.452+0000] {standard_task_runner.py:72} INFO - Started process 5989 to run task
[2024-12-11T13:06:27.448+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'instacart_etl', 'etl_dim_department', 'manual__2024-12-11T13:06:24.787823+00:00', '--job-id', '157', '--raw', '--subdir', 'DAGS_FOLDER/instacart_etl_dag.py', '--cfg-path', '/tmp/tmp1few2xe6']
[2024-12-11T13:06:27.453+0000] {standard_task_runner.py:105} INFO - Job 157: Subtask etl_dim_department
[2024-12-11T13:06:27.545+0000] {task_command.py:467} INFO - Running <TaskInstance: instacart_etl.etl_dim_department manual__2024-12-11T13:06:24.787823+00:00 [running]> on host 751be410a86b
[2024-12-11T13:06:27.711+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='instacart_etl' AIRFLOW_CTX_TASK_ID='etl_dim_department' AIRFLOW_CTX_EXECUTION_DATE='2024-12-11T13:06:24.787823+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-11T13:06:24.787823+00:00'
[2024-12-11T13:06:27.713+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-11T13:06:27.769+0000] {base.py:84} INFO - Retrieving connection 'mssql_default'
[2024-12-11T13:06:28.039+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-11T13:06:28.264+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-11T13:06:28.267+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=instacart_etl, task_id=etl_dim_department, run_id=manual__2024-12-11T13:06:24.787823+00:00, execution_date=20241211T130624, start_date=20241211T130627, end_date=20241211T130628
[2024-12-11T13:06:28.440+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-11T13:06:28.537+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-11T13:06:28.573+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
